include/sys/zstd/zstd.h

```diff
@@ -25,6 +25,10 @@
  * Copyright (c) 2016-2018, Allan Jude.
  * Copyright (c) 2018-2019, Sebastian Gottschall. All rights reserved.
  * Copyright (c) 2019-2020, Michael Niewöhner. All rights reserved.
+ * Copyright (c) 2020 The FreeBSD Foundation.
+ *
+ * Portions of this software were developed by Allan Jude
+ * under sponsorship from the FreeBSD Foundation.
```

Comment:
Same footer formatting might be nice here...

Reply:
well need another run of copyright formatting fixup anyways